src/onboarding/index.html

```diff
@@ -13,7 +13,7 @@
         <div class="tab1">
           <img src="./logo.svg" />
           <h1>
-            Welcome to<br/>
+            Welcome to<br />
```

Comment:
we should get rid of the first tab - showing the choice as fast as possible

Reply:
As I wrote above, no problem, but I will create another issue/pr.

Reply:
#4